recipes/spam/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "spam" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gricad-gitlab.univ-grenoble-alpes.fr/ttk/spam/-/archive/62a7c325dbea0d90314ba19a0ac76a61a23691f4/spam-62a7c325dbea0d90314ba19a0ac76a61a23691f4.tar
+  sha256: bc1c898392eac220784ae534550e2b3b0c071053f9aca870264cc79718b8886e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - numpy
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+    - pybind11
+    - eigen
+    - cgal-cpp
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+    - pyamg
+  host:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - pip
+    - numpy
+    - pybind11
+    - eigen
+    - cgal-cpp
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+  run:
+    - python       # [unix]
+    - python =3.7  # [win]
+    - matplotlib-base
+    - gmp                                    # [unix]
+    - m2w64-gmp                              # [win]
+    - mpir                                   # [win]
+    - r-base
```

Comment:
is it in fact the case that `r` is a hard requirement? might this be suited to multiple outputs, e.g. `libspam`, `spam-python` and `r-spam` or something, with a top-level `spam` metapackage?

Reply:
Yes `r` is a strong requirement as our package is not ready yet to be a metapackage.